x/faucet/handler.go

```diff
@@ -67,8 +67,8 @@ func handleMsgRequestCoins(ctx sdk.Context, keeper Keeper, msg types.MsgRequestC
 // Handle the add coins message and send coins from the signers account to the module account.
 func handleMsgAddCoins(ctx sdk.Context, keeper Keeper, msg types.MsgAddCoins) (*sdk.Result, error) {
 	if ctx.ChainID() != "sifchain" {
-		bank := keeper.GetBankKeeper()
-		err := bank.SendCoins(ctx, msg.Signer, types.GetFaucetModuleAddress(), msg.Coins)
+		supply := keeper.GetSupplyKeeper()
```

Comment:
Why the change in name? The bank keeper is still used, no?

Reply:
The refactor is just to mainitain consistence ,the request and add coins now use similar functions .